bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+    BSIP: T.B.D.
+    Title: Managed vesting balances
+    Authors: Fabian Schuh <Fabian.Schuh@BlockchainProjectsBV.com>
+             Stefan Schießl <Stefan.Schiessl@BlockchainProjectsBV.com>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-10-10
+    Discussion: <url>
```

Comment:
Please create a new Issue in BSIPs titled "Managed Vesting Balances" to capture further discussions on this topic. You may reference historical threads on forums, but suggest to have them locked prior to limit where new discussion is taking place.